tez-mapreduce/src/main/java/org/apache/tez/mapreduce/grouper/TezSplitGrouper.java

```diff
@@ -102,6 +103,20 @@ public abstract class TezSplitGrouper {
   public static final String TEZ_GROUPING_NODE_LOCAL_ONLY = "tez.grouping.node.local.only";
   public static final boolean TEZ_GROUPING_NODE_LOCAL_ONLY_DEFAULT = false;
 
+  /**
+   * Number of threads used to initialize the grouped splits, to asynchronously open the readers.
+   */
+  public static final String TEZ_GROUPING_SPLIT_INIT_THREADS = "tez.grouping.split.init-threads";
```

Comment:
this option should be in line with the other, both of them are about to configure the number of something, so:
```
tez.grouping.split.init.threads
tez.grouping.split.init.recordreaders
```
or:
```
tez.grouping.split.init.num-threads
tez.grouping.split.init.num-recordreaders
```

I prefer the first, we don't really use "num" in TezConfiguration either (only at 1 place), it's redundant I believe

Reply:
Ack!